README.md

```diff
@@ -38,7 +38,7 @@ require('telescope').setup{
         ["<C-s>"] = {
           before_action = function(selection) print("before C-s") end,
           action = function(selection)
-            vim.cmd.edit(selection.path)
+            vim.cmd("edit " .. selection.path)
```

Comment:
You changed more than described in your commit or PR. Why did you change the `vim.cmd` commands?

Reply:
Updated it myself :)